### PR TITLE
Setup env

### PR DESCRIPTION
**Pull request by spaceofmiah**
## Description
Setup working environment with react 

Fixes #(issue number 4) #4

## How Has This Been Tested?
This feature is well tested as running `npm start` launches web server and renders a react component without throwing any error or warning

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 barebones setup for JavaScript / Node projects replace this with a one-liner phrase describing this project or app
 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f17891ac33624e5a87b361bc915152bf)](https://app.codacy.com/gh/BuildForSDG/Team-108-Frontend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/Team-108-Frontend&utm_campaign=Badge_Grade_Settings)
 [![Codacy Badge](https://img.shields.io/badge/Code%20Quality-D-red)](https://img.shields.io/badge/Code%20Quality-D-red)
```
Comment (ezekieltech) [inline README.md:6]: We need to create this as an Issue so that we will have a standard README.md

File: package.json [hunk 2]
```diff
@@ -17,7 +17,9 @@
     "test:watch": "jest --watch",
```
Comment (ezekieltech) [inline package.json:17]: There's a quite some feedback from Codacy on this file. Many have to standard spacing and all. You should check it out.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang = "en">
+   <head>
+      <meta charset = "UTF-8">
```
Comment (ezekieltech) [inline public/index.html:4]: We may need more meta description.
`<meta name="viewport" content="width=device-width, height=device-height, initial-scale=1.0, minimum-scale=1.0">`
`<meta charset="utf-8">`
`<meta http-equiv="X-UA-Compatible" content="IE=edge">`
`<meta name="description" content="">`

**Review by ezekieltech — CHANGES_REQUESTED** [review]
The package.json file did not match the standards as prescribed by Codacy.

Also index.html file needs some additional meta description

**Review by Chukwuedo — APPROVED** [review]
No further comments

**Review by ezekieltech — APPROVED** [review]

**Review by Stephen-Kamau — APPROVED** [review]

**Pull request merged**